src/storm-dft/storage/dft/DFT.cpp

```diff
@@ -1176,6 +1187,45 @@ namespace storm {
             stream << "=========================================\n";
         }
 
+#ifdef STORM_HAVE_CARL
```

Comment:
Do we still use STORM_HAVE_CARL?

Reply:
We still use it in a lot of places. But this a future issue. I removed it here.